R/dependencies.R

```diff
@@ -1646,23 +1657,6 @@ renv_dependencies_report <- function(errors) {
 
 }
 
-renv_dependencies_confirm <- function(action, path, ...) {
-
-  withCallingHandlers(
```

Comment:
This was inlined into `renv_snapshot_dependencies()`, and that's now used consistently in each function that needs to compute a project's package dependencies.